website/docs/patterns/multi-select-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,63 @@
+
+![Total count of items and count of selected items](/assets/patterns/multi-select-patterns/total-count-selected-count.png)
+
+!!! Info
+
+Depending on how the data within a Table is fetched or if [cursor-based pagination](/components/pagination#numbered-vs-compact) is used, it may not be possible to get the total number of results in a data set.
+!!!
+
+## Bulk actions
+
+The act of selecting multiple results in a Table implies that the the user wants to edit, remove, or transform these results in some way. Consolidate these actions in a [List](/components/dropdown#list) toggled by a `small` [Dropdown](/components/dropdown) inline with the selected count, message, and total count.
+
+![Example of bulk actions](/assets/patterns/multi-select-patterns/bulk-actions.png)
+
+When working with instances of a Table where only a single action is necessary (e.g., bulk deleting a selection), use a `small` [Button](/components/button) in place of the Dropdown. We recommend using only `secondary` or `critical` Buttons in this scenario depending on the intent of the action.
+
+![Example of a single action](/assets/patterns/multi-select-patterns/single-action.png)
+
+### Editing results
+
+With a selection made, consider using a [Modal](/components/modal) to perform bulk edits or updates to parameters of the selected results. This is generally appropriate for editing five parameters or less, for more complex scenarios consider shifting the editing experience to a separate page.
+
+![Editing results in a Modal](/assets/patterns/multi-select-patterns/multi-select-edit-modal.png)
+
+### Deleting results
+
+Deleting or performing destructive actions on results should use an additional confirmation to ensure that only the intended results are affected. Use a `critical` [Modal](/components/modal) to communicate the number of results affected and what consequences might occur from deletion.
+
+![Deleting results in a Modal](/assets/patterns/multi-select-patterns/multi-select-delete-modal.png)
+
+## Bulk selection
+
+Use bulk selection to highlight additional, more complex methods of selecting rows in a Table. This type of selection should be global in scope, meaning that all results in the data set are affected, even those outside of the current view. Consolidate these functions in a Dropdown List paired with bulk actions.
+
+Some common examples of bulk selection include:
+
+- **Select all:** select all of the results in a data set.
+- **Deselect all/reset selection:** take the current selected results and deselect them.
+- **Select inverse:** deselect the currently selected results, select those that are currently deselected.
+- **Select odd/select even:** only select odd or even-numbered results; this can in part be determined by the current sorting mechanism.
```

Comment:
I don't see a realistic use case for this in our applications.
```suggestion
```

Reply:
+1 if a need arises later, we can always add this back in, but for now, let's stick to the use cases we know.